fair_drop/suspicious.py

```diff
@@ -0,0 +1,205 @@
+# See https://github.com/convex-labs/honestnft-shenanigans/issues/86 for more details
+# Given a collection of NFTs on OpenSea, detect suspicious NFTs
+# ! Note that this works only with NFT collections that have a friendly auto-incrementing IDs scheme
+import time
+from multiprocessing import Pool
+from argparse import ArgumentParser
+import logging
+
+import cloudscraper
```

Comment:
```suggestion
import requests
```
We don't need cloudscraper if we use a browser user-agent. This has the benefit of shaving of a few seconds and not relying on third party dependencies.

Reply:
:+1: